cmd/bake/bake.go

```diff
@@ -67,19 +70,74 @@ type bakePostRequest struct {
 	Wait bool   `json:"wait,omitempty"`
 }
 
+type repos struct {
+	URLs []string `yaml:"repos"`
+}
+
+type bakeStatus struct {
+	URL string
+	Err error
+}
+
 func run(opts *Options) error {
-	bodyPostReq := &bakePostRequest{
-		URL:  opts.URL,
-		Wait: opts.Wait,
+	var repos repos
+	uniqueURLs := make(map[string]int)
```

Comment:
I'm not sure about using a `map[string]int` over `map[string]bool`. Really, it's just a fast access list and we don't really care about the values, just the keys.

Maybe there's a more straight forward datastructure for this in Go?

Reply:
Changed it to ```map[string]bool```, makes more sense.

> Maybe there's a more straight forward datastructure for this in Go?
Not that I know of, wished Go had sets 😂, but the map will do.

Reply:
Yeah, `map[type]bool` is about as close as we can get 🥲